lib/orbital/models/response.rb

```diff
@@ -187,6 +187,11 @@ def self.search_where_clause(t, search_key)
 
         return where_clause
       end
+
+      # not needed, put a big number beyond the accurate limit (20k) here to make the pagination work
```

Comment:
Confused by the comment; if it is "not needed", why we have it :)?

Reply:
Because in the `::Killbill::Plugin::ActiveMerchant::ActiveRecord::Response`, it uses the min of `max_nb_records` and input `limit` to decide the page size.

Setting `max_nb_records` to a large number would basically make it fine with normal `limit`s.